app/lib/common/widgets/side_sheet.dart

```diff
@@ -56,6 +56,7 @@ class SideSheet extends StatelessWidget {
         margin: EdgeInsets.only(
           bottom: MediaQuery.of(context).viewInsets.bottom,
         ),
+        padding: const EdgeInsets.only(top: 12),
```

Comment:
maybe add a code comment so we understand why this is here?
```suggestion
        padding: const EdgeInsets.only(top: 12), // keep space to top bar to prevent us being covered by front-camera etc..
```